server/src/main/java/org/apache/druid/metadata/segment/cache/Metric.java

```diff
@@ -47,6 +47,11 @@ private Metric()
   // CACHE METRICS
   private static final String METRIC_NAME_PREFIX = "segment/metadataCache/";
 
+  /**
+   * Number of transactions performed on the cache for a datasource.
+   */
+  public static final String TRANSACTION_COUNT = "transactions";
```

Comment:
Please add the metrics in the metrics.md doc.

Reply:
added.